### PR TITLE
Fix #2545, bring redshift wrapIdentifier custom hook up to date with postgres

### LINKED ISSUE #2545: Redshift woes
have had to switch from postgres to redshift, but redshift doesn't like one of my migration files:

using knex 0.14.3

```
var config = require('../../config')

exports.up = function (knex, Promise) {
  return knex.schema.withSchema(config.get('/db/schema/metrics')).table('homenetstats', function (t) {
    t.string('iLotId').nullable()
    t.string('iLotName').nullable()
    t.dateTime('marketUpdatedDate').nullable()

    t.index('iLotId')
    t.index('iLotName')
    t.index('marketUpdatedDate')
  })
}

exports.down = function (knex, Promise) {

}
```

Knex:warning - migration failed with error: alter table "dev_temp"."homenetstats" add column "iLotId" varchar(255) null, add column "iLotName" varchar(255) null, add column "marketUpdatedDate" timestamptz null - syntax error at or near ","
error: syntax error at or near ","
    at Connection.parseE (/Users/emmett/dev/vroom/aggregated-inv-svc/node_modules/pg/lib/connection.js:545:11)
    at Connection.parseMessage (/Users/emmett/dev/vroom/aggregated-inv-svc/node_modules/pg/lib/connection.js:370:19)
    at TLSSocket.<anonymous> (/Users/emmett/dev/vroom/aggregated-inv-svc/node_modules/pg/lib/connection.js:113:22)
    at emitOne (events.js:115:13)
    at TLSSocket.emit (events.js:210:7)
    at addChunk (_stream_readable.js:250:12)
    at readableAddChunk (_stream_readable.js:237:11)
    at TLSSocket.Readable.push (_stream_readable.js:195:10)
    at TLSWrap.onread (net.js:586:20)

### PR DESCRIPTION
**Pull request by acofer**

### CODE REVIEW

File: test/integration/builder/additional.js [hunk 1]
```diff
@@ -175,6 +175,9 @@ module.exports = function(knex) {
           // Insert new data after truncate and make sure ids restart at 1.
           // This doesn't currently work on oracle, where the created sequence
           // needs to be manually reset.
+          // On redshift, one would need to create an entirely new table and do
```
Comment (elhigu) [inline test/integration/builder/additional.js]: This block is in a bit strange position separating comment about oracle from if()... maybe skipping this for oracle could be also implemented the same way that is is implemented for redshift.

**Comment by acofer** [comment]
I'm not sure why the travis builds are failing for node < 7. I assume it's some sort of babel misconfiguration, but I don't know what? I have `nvm`, let me know if I should try to reproduce locally.

**Comment by elhigu** [comment]
Looks like it is eslint that is failing. Maybe that package was updated or something... I'll try to rerun build.

**Comment by elhigu** [comment]
Yup, it was just a glitch in the matrix. Thankes !

**Pull request merged**